packages/rocketchat-crowd/server/settings.js

```diff
@@ -14,6 +14,7 @@ Meteor.startup(function() {
 		this.add('CROWD_Sync_User_Data', false, { type: 'boolean', enableQuery, i18nLabel: 'Sync_Users' });
 		this.add('CROWD_Sync_Interval', 'Every 60 mins', { type: 'string', enableQuery: enableSyncQuery, i18nLabel: 'Sync_Interval', i18nDescription: 'Crowd_sync_interval_Description' });
 		this.add('CROWD_Clean_Usernames', true, { type: 'boolean', enableQuery, i18nLabel: 'Clean_Usernames', i18nDescription: 'Crowd_clean_usernames_Description' });
+		this.add('CROWD_Allow_Custom_Username', true, { type: 'boolean', public: true, i18nLabel: 'CROWD_Allow_Custom_Username' });
```

Comment:
This setting doesn't need to be public since it's used only at the server side

Reply:
Removed public property.